gn3/api/search.py

```diff
@@ -212,6 +212,7 @@ def parse_query(synteny_files_directory: Path, query: str):
     queryparser.add_boolean_prefix("peakchr", "XPC")
     queryparser.add_prefix("description", "XD")
     queryparser.add_prefix("rif", "XRF")
+    queryparser.add_prefix("wikidata", "XRWD")
```

Comment:
Please make this "wiki"; and similarly: "XWK":
```suggestion
    queryparser.add_prefix("wiki", "XWK")
```

Reply:
Done. I've also made the corresponding change in the indexer script.